docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,220 @@
+# AMWA BCP-XXX-YY: EDID Mapping in NMOS Systems \[Work In Progress\]
+
+
+## Background
+
+AMWA NMOS specifications provide resources for specifying receiver capabilities of an NMOS device.  Recent extension of these resources allow NMOS Controllers developed for PRO A/V User-Stories to be created.  In particular, these NMOS Controllers include Controllers developed for the AIMS sponsered IPMX set of open standards [IPMX](#ipmx-ref). 
+
+## Scope
+
+This document provides Best Current Practice guideluines for how to implement mappings of EDID elements onto AMWA IS-04 and IS-EDID-CM resources.  The document also provides guidance on how implementers may use the APIs to achieve some common IPMX type User-Stories and associated use-cases as described in the EDID Connection Management Architecture [EDID-CM-ARCH](#edid-cm-arch-ref).
+
+## Use of Normative Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](#rfc2119-ref).
+
+## Definitions
+
+_See also the [NMOS Glossary](https://github.com/AMWA-TV/nmos/wiki/Glossary), and definitions within RFCs._
+
+
+### IPMX 
+  The IPMX (Internet Protocol for Media Experience) is a set of open standards and specifications to enable the carriage of compressed and uncompressed video, audio and data over IP networks. It includes provisions for control, copy protection, connection management and security. Includes use of SMPTE and NMOS standards.
+
+
+### IPMX Controller
```

Comment:
We tended to get rid of `IPMX Controller`, `IPMX Device Control` and combinations. `IPMX Device Controller`, `IPMX Device Control API` and `IPMX Control System` were suggested in #14 and implemented in #23.